config/dev-vm.js

```diff
@@ -56,10 +56,15 @@ module.exports = merge(base, {
 		federationService: {
 			uri: 'https://marketplace-api.dk1.kiva.org/graphql'
 		},
-		paypal : {
+		paypal: {
 			url: 'www.sandbox.paypal.com',
 			environment: 'sandbox'
 		},
+		oneTrust: {
+			enable: true,
+			key: 'db9dcf94-1c32-40fb-9a57-cefafea1088d',
+			domainSuffix: '-test',
```

Comment:
The only difference between the oneTrust test & production scripts is the domainSuffix - for production this block will look like: 
```
			enable: true,
			key: 'db9dcf94-1c32-40fb-9a57-cefafea1088d',
			domainSuffix: '',
```